GarbageCollectionApp/GarbageCollectionApp/Views/Admin/CollectionMap.cshtml

```diff
@@ -0,0 +1,192 @@
+            <text>
+                        {
+                            binId: '@item.IdGarbageBin',
+                            time: '@collectionTime.ToString("yyyy-MM-ddTHH:mm:ss")',
+                            address: '@Html.Raw(item.Address.Replace("'", "\\'"))',
+                            lat: @lat.ToString(System.Globalization.CultureInfo.InvariantCulture),
+                            lng: @lng.ToString(System.Globalization.CultureInfo.InvariantCulture),
+                            timestamp: new Date('@collectionTime.ToString("yyyy-MM-ddTHH:mm:ss")').getTime()
+                        },
+            </text>
+        }
+            ].sort((a, b) => a.timestamp - b.timestamp);
+
+            const markers = [];
+            const routeSegments = [];
+            let previousPoint = null;
+
+            collectionPoints.forEach((point, index) => {
+                if (typeof point.lat === 'number' && typeof point.lng === 'number') {
+                    // Create marker with sequence number
+                    const marker = L.marker([point.lat, point.lng]).addTo(map);
+                    const sequenceNumber = L.divIcon({
+                        html: `<div style="
+                            background: #1976d2;
+                            color: white;
+                            border-radius: 50%;
+                            width: 24px;
+                            height: 24px;
+                            display: flex;
+                            align-items: center;
+                            justify-content: center;
+                            font-weight: bold;
+                            border: 2px solid white;
+                        ">${index + 1}</div>`,
+                        className: 'sequence-marker',
+                        iconSize: [24, 24]
+                    });
+
+                    L.marker([point.lat, point.lng], {icon: sequenceNumber}).addTo(map);
+
+                    // Format the date properly
+                    const dateObj = new Date(point.time);
+                    const formattedTime = isNaN(dateObj.getTime()) ?
+                        'Time not available' :
+                        dateObj.toLocaleString('en-US', {
+                            weekday: 'short',
+                            year: 'numeric',
+                            month: 'short',
+                            day: 'numeric',
+                            hour: '2-digit',
+                            minute: '2-digit'
+                        });
+
+                    marker.bindPopup(`
+                        <b>Stop #${index + 1}</b><br>
+                        <b>Bin ID:</b> ${point.binId}<br>
+                        <b>Time:</b> ${formattedTime}<br>
+                        <b>Address:</b> ${point.address}
+                    `);
+                    markers.push(marker);
+
+                    // Create line segment if we have a previous point
+                    if (previousPoint) {
```

Comment:
Are sens sa avem linie din pin in pin? Pare ca sunt trase linile prin casele oamenilor, sunt 100% sigur ca nu pe acolo a mers masina de colectare 😄 
![image](https://github.com/user-attachments/assets/611caed2-a336-40a0-8080-0a058ea9317a)